gravitee/templates/ui-deployment.yaml

```diff
@@ -18,6 +18,7 @@ spec:
     metadata:
       annotations:
         chaos.alpha.kubernetes.io/enabled: "{{ .Values.chaos.enabled }}"
+        checksum/config: {{ include (print $.Template.BasePath "/ui-configmap.yaml") . | sha256sum }}
```

Comment:
+1

Reply:
Yeah, that's the way I found to force the deployment to pick up a change in configmap.